apps/bare-expo/android/app/src/main/java/dev/expo/payments/MainApplication.kt

```diff
@@ -23,7 +25,10 @@ class MainApplication : Application(), ReactApplication {
       override fun getPackages(): List<ReactPackage> {
         // Packages that cannot be autolinked yet can be added manually here, for example:
         // packages.add(new MyReactNativePackage());
-        return PackageList(this).packages
+        return PackageList(this).packages.apply {
+          add(BenchmarkingPackage())
+          add(BenchmarkingTurboPackage())
+        }
```

Comment:
could you add a local config plugin for this? that would help bare-expo to regenerate native files from expo prebuild in the future?
an example at https://github.com/expo/expo/blob/b642518e8adc904b440be44d99357b159637bfd9/apps/bare-expo/app.json#L91